docs/application/dotnet/guides/nui/animation.md

```diff
@@ -123,27 +123,27 @@ During the playback, you can receive notifications at various stages of the anim
     animation.ProgressReached += progressReachedCallback;
     ```
 
--   `Finished`: Notify when the animation finishes
+-   `Finished`: Notifies when the animation finishes.
     ```csharp
     animation.Finished += animationFinishedCallback;
     ```
 
 <a name="alphafunctions"></a>
 ## Alpha Functions
 
-In animations, alpha functions are used to specify the an animation progresses over time. This allows the animation to be, for example, accelerated, decelerated, repeated, or bounced.
+In animations, alpha functions are used to specify an animation progresses over time. The alpha function allows the animation to be accelerated, decelerated, repeated, or bounced.
 
 ### Built-in Alpha Function
 
 The simplest way to define an alpha function is to use `built-in alpha function` which NUI provides for the convenience.
```

Comment:
Please make this small change **which** to **that**:

The simplest way to define an alpha function is to use `built-in alpha function` that NUI provides for the convenience.

@prasadam-p, I have one more query, is there any specific reason to call out `built-in alpha function` in code tag? Because I see the ENUM value is mentioned differently, and so is in the code. Please clarify my doubt.

Reply:
I removed code tag from `built-in alpha function`